cpp/ql/test/query-tests/Security/CWE/CWE-190/semmle/uncontrolled/test.c

```diff
@@ -74,7 +74,7 @@ void randomTester() {
   {
     int r = RAND2();
 
-    r = r - 100; // BAD
+    r = r - 100; // BAD [NOT DETECTED]
```

Comment:
In the final merged version this (and line 83) is marked `[NOT DETECTED]` but the results are there in the `.expected`.

Reply:
Good catch!